src/main/java/com/bankxpvalue/BankXpValueConfig.java

```diff
@@ -24,17 +24,27 @@ default boolean showTutorial(){
     void setTutorial(boolean tutorial);
 
     @ConfigItem(
-            position = 2,
+        position = 2,
+        keyName = "createInCenter",
+        name = "Create overlay in center",
+        description = "Creates the overlay in the center of the bank window"
```

Comment:
I don't understand what this is supposed to do exactly. You said it `lets the overlay position not reset on show`, but that has nothing to do with this config description... Despite that, I don't see the difference between this and the config setting I had for locking the overlay in the center.

Reply:
For sure. So it "lets the overlay position not reset on show" when you set it to false. I am defaulting it to `true` on line 33 just so that current behavior is maintained. It's similar to the `keepFixed` config, but this is just controlling whether it is set to the center `onShow`. In this case, if you have `keepFixed == true` then this setting does nothing. But with `keepFixed == createInCenter == false` then when I toggle the overlay, it remains where I left it.

If you can think of a more concise configuration, let me know. I just wanted to not touch existing code as much as possible.

Reply:
Ohhh I see I see. I like that. Instead of making a new menu entry, I think having it be the default behavior when `keepFixed == false` would be optimal. What do you think?

Reply:
I think that makes sense. Having it appear in the center every time it is opened (when not keeping it fixed) isn't that useful or helpful in my opinion. I'll make the change!

Reply:
Hey I was just thinking....is keepFixed really even desired? Like, I see trying to drag the overlay away and it snapping back as an annoyance. It seems like the only options we'd really need are "appear in last location" and "appear in center". Doing it that way would simplify the code some I think, and allow the user to be able to drag the overlay outside of the bank temporarily. Thoughts?

Reply:
Hmmm, I see what you're saying. I think having it in the center should always be the default position (assuming the user does _not_ want to move it), otherwise, it would appear awkwardly on the screen somewhere. If you can have it so that the center is the default position, I'm all for making those changes!